package.json

```diff
@@ -18,7 +18,10 @@
   "types": "types/index.d.ts",
   "files": [
     "index.js",
-    "types/index.d.ts"
+    "cache.js",
```

Comment:
can only the following be added `"dist"`?

Reply:
alright there is no  bundling happening, so no dist folder would be generated. please use

"*.js" so we won't need to remember to update this again.